core/common/src/thrift/block_master.thrift

```diff
@@ -18,6 +38,14 @@ struct GetBlockInfoTResponse {
   1: common.BlockInfo blockInfo
 }
 
+struct GetBlockMasterInfoTOptions {
+ 1: set<BlockMasterInfoField> filter
```

Comment:
@aaudiber would it be useful to make these optional?

Reply:
The filter is optional in our usage.

users can pass in null to get all the BlockMasterInfo.
BlockMasterInfo info = client.getBlockMasterInfo(null);

because in BlockMasterClientServiceHandler.getBlockMasterInfo
we have:
        for (BlockMasterInfoField field : options.getFilter() != null ? options.getFilter()
            : Arrays.asList(BlockMasterInfoField.values())) {